Codecademy/gamut
# feat(Border Radius): Fixing Atoms' border radii

### Overview
Addressing the atoms first, fixing border radii
See notion for more details: https://www.notion.so/codecademy/Border-Radius-Tracker-55c1c9e9ab49480995d314ac3fb784ff

### PR Checklist

- [ ] Related to a mixture of JIRA tickets: [GM-592](https://skillsoftdev.atlassian.net/browse/GM-592), [GM-892](https://skillsoftdev.atlassian.net/browse/GM-829), [GM-828](https://skillsoftdev.atlassian.net/browse/GM-828), [GM-830](https://skillsoftdev.atlassian.net/browse/GM-830)

#### Testing Instructions

1. Visit the notion page to see a list of changed components, go to their relative paths and check that things render the same and work the same way. 

[GM-592]: https://skillsoftdev.atlassian.net/browse/GM-592?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[GM-828]: https://skillsoftdev.atlassian.net/browse/GM-828?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[GM-830]: https://skillsoftdev.atlassian.net/browse/GM-830?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
